cdc/scheduler/internal/v3/replication/replication_set.go

```diff
@@ -374,16 +487,17 @@ func (r *ReplicationSet) pollOnPrepare(
 				zap.Stringer("tableState", input),
 				zap.String("captureID", captureID),
 				zap.Any("replicationSet", r))
-			r.Primary = ""
-			delete(r.Captures, captureID)
+			r.clearPrimary()
 			return nil, false, nil
-		} else if r.Secondary == captureID {
+		} else if r.isInRole(captureID, CaptureRoleSecondary) {
```

Comment:
`else if` can be simplified to `if` directly.

Reply:
I am planning to add a new role in later PR, called `Standby` for handling unplanned shutdown.